protocol/messenger.go

```diff
@@ -4117,16 +4117,11 @@ func (m *Messenger) markAllRead(chatID string, clock uint64, shouldBeSynced bool
 	return m.persistence.SaveChats([]*Chat{chat})
 }
 
-func (m *Messenger) MarkAllRead(chatID string) error {
-	notifications, err := m.persistence.DismissAllActivityCenterNotificationsFromChatID(chatID, m.getCurrentTimeInMillis())
+func (m *Messenger) MarkAllRead(ctx context.Context, chatID string) error {
+	_, err := m.DismissAllActivityCenterNotificationsFromChatID(ctx, chatID, m.GetCurrentTimeInMillis())
```

Comment:
I case we change the state of some notifications, i think we should add those notifications to the response for the client